src/style.css

```diff
@@ -0,0 +1,49 @@
+* {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+  box-sizing: border-box;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+nav {
+  display: flex;
+}
+
+a {
```

Comment:
better to apply such styles to `class` instead of `tag`. You could have many `<a>` tags. Even here you will have 2 different types of links. One with a logo inside, another with text. Figure it out.